src/jsutils/ObjMap.ts

```diff
@@ -1,13 +1,13 @@
 export interface ObjMap<T> {
-  [key: string]: T;
+  [key: string | symbol]: T;
```

Comment:
It would be nice if we didn’t have to change this type as it’s used very often in the code base and I think we only want to support symbols on extensions

Reply:
We use `toObjMap` for extensions though, let me see what I can do